packages/errors/src/errors/ResponseError.ts

```diff
@@ -65,29 +68,36 @@ export class ResponseError extends Error {
   ): Promise<ResponseError> {
     const data = await parseErrorResponseBody(response);
 
-    const status = data.response.statusCode || response.status;
-    const statusText = data.error.name || response.statusText;
```

Comment:
Curious about this one. It used to be that we wanted to actually propagate custom error names from the backend to the frontend. I could understand if we for example treated `'Error'` like undefined, but entirely removing this feature - is that the right move?